src/status_im2/setup/dev.cljs

```diff
@@ -4,7 +4,18 @@
             [status-im.ethereum.json-rpc :as json-rpc]
             ["react-native" :refer (DevSettings LogBox)]))
 
-(.ignoreAllLogs LogBox)
+;; Only ignore warnings/errors that cannot be fixed for the time being.
+;; When you add a warning to be ignored explain below why it is ignored and how it can be fixed.
+;; When a warning is fixed make sure to remove it from here.
+(.ignoreLogs ^js LogBox (clj->js ["undefined is not an object (evaluating 'e.message')"
```

Comment:
please no, it's not worth it, there are always will be annoying warnings when hot reload, etc

Reply:
These warnings are meant to be fixed :) Otherwise, we will be piling up warnings and errors again. And this command does not just ignore warnings but all errors, which can cause bugs. For example, someone wrote `ellipsizeMode` inside a style prop, but this is incorrect, `ellipsizeMode` is a prop itself. Another one, lists should always be using unique keys for performance reasons, and also the keys must be strings. Also, some re-frame events had no effects implemented. IgnoreAllLogs is going to ignore these things and more. So it is better to see the warnings. When you see a warning/error you can just add it to the LogBox if you do not wish to fix it right now, at least we would know there is something being ignored. :)

Reply:
i mean this won't work during development, because you will have warnings and errors and that's fine, but you don't want to hide them manually all the time

Reply:
I agree with @OmarBasem, it's better not to sweep warnings under the carpet, so we can detect and fix them.

Reply:
i also agree, we can see them in the log, but during active development i don't want to see temporary warnings  in the app

Reply:
ok if you really want this we can try it, but at least we need to ignore re-frame hot reload warnings

Reply:
@flexsurfer We shouldn't be having warnings and errors all the time. After this PR you will not find many warnings, I have fixed all the warnings that are appearing. Whenever there is a new warning/error, we either fix it right now or add it to the ignore box. Simple. And if someone really wishes they can toggle `ignoreAllLogs` while developing, tho it is better not to.

Reply:
I am currently not getting any re-frame warnings

Reply:
Btw, since RN v0.63 you do not get the warning screen in your face, you only get a small notification at the bottom of the screen which is much less intrusive

Reply:
I was trying to remove the in-app error overlay early today. 
So If I understand right, `.ignoreAllLogs` and `.ignoreLogs` only prevent things to display in the error overlay in the emulator, right?  We can still see them in the terminal.

Reply:
As of now in `develop` all logs are ignored. If you are getting an error overlay, this means that it is a fatal error. These ones cannot be disabled. And you can still see all logs in the terminal.

Reply:
OK, got that. Thanks. 

> Btw, since RN v0.63 you do not get the warning screen in your face, you only get a small notification at the bottom of the screen which is much less intrusive

(Not related to this PR) If only we can apply this to fatal error as well. It's really annoying since I hadn't found a way to minimize that without mouse.

Reply:
@yqrashawn they are called fatal for a reason :) I do not think there is a way to disable them in debug mode. In release mode, they would crash the app.

Reply:
Yes, it's just it's the same error again and again. I navigate the app by evaluating code in the editor. So, I can just navigate back and forth to check if the error fixed. The error overly is just in my way to quickly try things.

Reply:
@OmarBasem when you hot reload re-frame events there will be a warnings about overwriting events

Reply:
@flexsurfer I just tried this and I am not getting any warnings. If we start getting annoying `re-frame` warnings, then for sure will add it 👍

Reply:
did you try changes in the event handler? or fx ?

Reply:
Yes, I tried doing changes to `navigate-to-chat` fx in `status-im.chat.models` and also tried doing changes to `status-im2.setup.events` and no warnings happened

Reply:
hm strange, ok